packages/paste-website/src/pages/layout/media-object/index.mdx

```diff
@@ -0,0 +1,287 @@
+---
+title: Media Object - Layout
+description: Media Object is a layout utility that you can't unsee.
+slug: /layout/media-object/
+---
+
+import {graphql} from 'gatsby';
+import {Box} from '@twilio-paste/box';
+import {Paragraph} from '@twilio-paste/paragraph';
+import {Alert} from '@twilio-paste/alert';
+import {Menu, MenuButton, SubMenuButton, MenuItem, MenuGroup, MenuSeparator, useMenuState} from '@twilio-paste/menu';
+import {MediaObject, MediaFigure, MediaBody} from '@twilio-paste/media-object';
+import {ChevronDownIcon} from '@twilio-paste/icons/esm/ChevronDownIcon';
+import {subMenuExample} from '../../../component-examples/MenuExamples.ts';
+import {SidebarCategoryRoutes} from '../../../constants';
+import Changelog from '@twilio-paste/media-object/CHANGELOG.md';
+
+export const pageQuery = graphql`
+  {
+    allPasteLayout(filter: {name: {eq: "@twilio-paste/media-object"}}) {
+      edges {
+        node {
+          name
+          description
+          status
+          version
+        }
+      }
+    }
+    mdx(fields: {slug: {eq: "/media-object/"}}) {
+      fileAbsolutePath
+      frontmatter {
+        slug
+        title
+      }
+      headings {
+        depth
+        value
+      }
+    }
+  }
+`;
+
+<ComponentHeader
+  name="Media Object"
+  categoryRoute={SidebarCategoryRoutes.LAYOUT}
+  githubUrl="https://github.com/twilio-labs/paste/tree/main/packages/paste-core/layout/media-object"
+  storybookUrl="https://twilio-labs.github.io/paste/?path=/story/layout-stack--stack"
+  data={props.data.allPasteLayout.edges}
+/>
+
+---
+
+<contentwrapper>
+
+<PageAside data={props.data.mdx} />
+
+<content>
+
+## Guidelines
+
+### About Media Object
+
+First created by [Nicole Sullivan](http://www.stubbornella.org/content/2010/06/25/the-media-object-saves-hundreds-of-lines-of-code/) back in 2010, the Media Object in Paste aims to serve many of the same purposes as the original concept. It's a layout pattern that you will see all across the web, on almost all the websites you will come across. We're pretty sure you'll never unsee this going forward.
+
+The concept is this; there is an image or figure positioned horizontally next to some form of content. The figure can sit on either or both sides of the content. That's it. It sounds dumb, but it's literally everywhere. Even in the top of this website with the Paste logo.
```

Comment:
```suggestion
**So what's a media object?** An image or figure positioned horizontally next to some form of content. The figure can sit on either or both sides of the content. That's it. It sounds really simple, but it's literally everywhere on the Internet. For example, the top left of this website uses a Media Object.
```

Reply:
Can we add a little code snippet here to show the paste design system media object example?